### PR TITLE
Misc fixes to diffusers blog post

### PR DESCRIPTION
**Pull request by osanseviero**
It still needs a couple of updates at the end

### CODE REVIEW

**Review by pcuenca — APPROVED** [review]
Thanks a lot for bringing this up to date, @osanseviero 🙌 I'm taking a look at the final step.

File: stable_diffusion.md [hunk 1]
```diff
@@ -503,8 +495,11 @@ And finally, let's convert the image to PIL so we can display or save it.
 
 
 ```python
-image = (image / 2 + 0.5).clamp(0, 1)
-image = image.detach().cpu().permute(0, 2, 3, 1).numpy()
+from PIL import Image
+
+# TODO: update, these are np arrays now, not torch tensors
+image = (image / 2 + 0.5).clip(0, 1)
+#image = image.detach().cpu().permute(0, 2, 3, 1).numpy()
```
Comment (pcuenca) [inline stable_diffusion.md:502]:
```suggestion
image = (image / 2 + 0.5).clip(0, 1)
image = image.detach().cpu().permute(0, 2, 3, 1).numpy()
```

Hmm, `image` comes from the `vae.decode(latents).sample` step above, so they really are tensors. Let me know if I'm missing anything, @osanseviero :)

Also, `from PIL import Image` was done earlier when the grid function was defined.

**Review by pcuenca — APPROVED** [review]

**Comment by osanseviero** [comment]
(unrelated, but I also saw reports such as https://huggingface.co/CompVis/stable-diffusion-v1-4/discussions/165#639646f3620fc4300bb4b68e so there are other places with similar things)

**Comment by osanseviero** [comment]
> Hmm, image comes from the vae.decode(latents).sample step above, so they really are tensors. Let me know if I'm missing anything, @osanseviero :)

Mmm yes, you're right, please ignore that feedback

**Comment by patrickvonplaten** [comment]
Thanks a lot @osanseviero ! 

Fixed: https://huggingface.co/CompVis/stable-diffusion-v1-4/discussions/165#639646f3620fc4300bb4b68e directly on "main"
